### ISSUE TITLE
Mixed cerebral palsy

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7300]
**Preferred term label:**
Mixed cerebral palsy

**Synonyms**
Mixed CP

**Definition (free text, please give PubMed ID)**
A type of cerebral palsy characterized by the presence of combinations of features such as spasticity, choreoathetosis, ataxia, dyskinesia, hemiplegia, diplegia and quadriplegia (PMID: 16761450; 32606703; 2148667).

**Parent term (use hpo.jax.org/app)**
Cerebral palsy HP:0100021

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
@jagcsun I think this is a disease and should be added to MONDO. There is a grey zone between MONDO and HPO that is a little difficult, and currently, Cerebral palsy is a term since it is regarded as a feature of some Mendelian diseases, but we should actually strive to describe the individual phenotypic features. Let's plan on working with @nicolevasilevsky to build out the CP branch of MONDO once we have finished the review!

**Comment by nicolevasilevsky** [comment]
We have this term in Mondo: MONDO_0000400

I'd love to work with you two on the CP branch. :)

**Comment by nicolevasilevsky** [comment]
Hi @peupeubangbang we should love advice on the CP branch in Mondo, thank you! Let me know how we can best coordinate.

**Comment by peupeubangbang** [comment]
Hi @nicolevasilevsky I've been mapping MONDO and HPO to the minimum data elements for CP type & topography held on the Australian Register (which aligns closely with the European register). 

Once finished I'll share my effort with you @pnrobinson @jagcsun to see what you think? Then we can all have a chat including Gareth. The experts tell me hypotonic CP will be difficult for HPO (may need child terms introduced) but we have expertise from Oz & Europe we can lean on for classification.

**Comment by nicolevasilevsky** [comment]
Hi @peupeubangbang - this sounds great, please do let me know when you are done and are ready to chat.
